src/Chirp.Web/Pages/Shared/_CheepListPartial.cshtml

```diff
@@ -10,17 +10,23 @@
             {
             <li>
                 <p>
-                    <strong>
-                        <a href="/@cheep.Author.Id">@cheep.Author.Name</a>
-                    </strong>
+                    @if(@cheep.Author.Name.Equals("[Deleted user]")) {
```

Comment:
It would probably be better to have the logic of deleted/not deleted on the model eller user, instead of checking "business logic" in front end?

Edge case -> if a user for some reason has the name [Deleted user], other users will not be able to access their usertimeline.